libs/product/src/reducers/product-entities.reducer.ts

```diff
@@ -1,33 +1,33 @@
 import { createEntityAdapter, EntityState, EntityAdapter, Dictionary } from '@ngrx/entity';
 
-import { ProductGridActionTypes, ProductGridActions } from '../actions/product-grid.actions';
-import { ProductActionTypes, ProductActions } from '../actions/product.actions';
```

Comment:
DaffProductActions?

Reply:
Addressed in 12b6a9f55d655fd6d185281b323c6d341f1f9a9e